spa/src/components/paymentProviders/stripe/stripeFns.test.ts

```diff
@@ -147,7 +147,7 @@ describe('getPaymentSuccessUrl', () => {
     expect(search.get('pageSlug')).toEqual(args.pageSlug);
     expect(search.get('rpSlug')).toEqual(args.rpSlug);
     expect(search.get('fromPath')).toEqual(args.pathName === '/' ? '' : args.pathName);
-    expect(search.get('payment_intent_client_secret')).toEqual(args.stripeClientSecret);
+    expect(search.get('contributionUuid')).toEqual(args.contributionUuid);
```

Comment:
```suggestion
    expect(search.get('contributionUuid')).toBe(args.contributionUuid);
```

I think since it's a plain string, we can use the stronger comparison. `toEqual()` is good when asserting on objects.